include/rounding.h

```diff
@@ -16,7 +16,7 @@
 
 
 #include "khach.h"
-
+#include "../external/Eigen/Eigen"
```

Comment:
We don't have to include it here because the same header file is used in the CRAN package